test/test_suite_test.go

```diff
@@ -183,3 +184,23 @@ func getOrSetEnvVar(key, value string) {
 		os.Setenv(key, value)
 	}
 }
+
+var _ = AfterEach(func() {
+	path, err := helpers.ReportDirectory()
+
+	if err != nil {
+		log.Errorf("cannot create ReportDirectory")
+		return
+	}
+
+	defer config.TestLogWriterReset()
```

Comment:
oh, I meant also all the file writing. We're basically doing a flush and reset operation and we don't expect to not do them all together. Anyway, whatever you want is fine.